content/developer/tutorials/server_framework_101/04_relational_fields.rst

```diff
@@ -646,6 +646,11 @@ to a list of offers received from potential buyers.
    #. Allow connecting properties to multiple offers.
    #. Modify the form view of properties to display offers in a new notebook page titled "Offers".
 
+   .. tip::
+      The `default` field argument expects a callable function, not a precalculated value. If you
```

Comment:
I prefer to limit the number of "we'll see that later" occurrences because they don't really help the user. Their only purpose is to say "it's ok if you don't understand right away", but here the user should be able to understand, in particular with the help of the solutions.